xilem/examples/flex.rs

```diff
@@ -20,7 +20,7 @@ fn big_button(
 }
 
 fn app_logic(data: &mut i32) -> impl WidgetView<i32> + use<> {
-    flex_row((
+    row((
```

Comment:
As it's not immediately obvious anymore with that name, maybe a comment above:

```suggestion
    // This is the flex view, alternatives are `column` or `flex` which allows dynamically switching the axis
    row((
```